shared/wallets/wallet/settings/popups/rename-account/container.js

```diff
@@ -37,7 +42,7 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => ({
 
 export default compose(
   withStateHandlers(
-    {name: ''},
+    {name: '__INITIAL_PLACEHOLDER_NAME__'},
```

Comment:
you can just init this with the name. 
```
{name: props => props.name}
```

Reply:
i assumed the props wouldn't be available there - don't know why I didn't just try it.

Reply:
that didn't work. I changed it to use `null`, since `''` wouldn't work.